Paco-Server/src/com/google/sampling/experiential/server/CloudSqlSearchServlet.java

```diff
@@ -158,11 +159,14 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
           SearchUtil.addJoinClause(clientJsqlStatement);
         }
         if(sqlQueryObj.isFullEventAndOutputs()) {
-          optimizedSelect = modifyToOptimizePerformance(clientJsqlStatement);
+          boolean outputColsInWhere = false;
```

Comment:
Small suggestion: these three statements could be one statement.
Original
---
 boolean outputColsInWhere = false;
 +          if (sqlQueryObj.getCriteriaQuery().contains(OutputBaseColumns.ANSWER) || sqlQueryObj.getCriteriaQuery().contains(OutputBaseColumns.NAME)) {
 +            outputColsInWhere = true;
 +          }

suggestion =>

 boolean outputColsInWhere = sqlQueryObj.getCriteriaQuery().contains(OutputBaseColumns.ANSWER) ||    
      sqlQueryObj.getCriteriaQuery().contains(OutputBaseColumns.NAME));